cmd/run.go

```diff
@@ -56,6 +56,10 @@ func addFailOnNotFoundFlag(cmd *cobra.Command) {
 	cmd.Flags().BoolVarP(&FailOnNotFound, "fail-on-not-found", "f", true, "Fail if given models are not found")
 }
 
+func addEnableSchemaBasedTestsFlag(cmd *cobra.Command) {
+	cmd.Flags().BoolVarP(&EnableSchemaBasedTests, "enable-schema-based-tests", "f", false, "Enable Schema-based tests")
```

Comment:
Looks like the flag "f" is already being used by the fail-on-not-found option, you might want to pick another flag for it, something like:
```suggestion
	cmd.Flags().BoolVarP(&EnableSchemaBasedTests, "enable-schema-based-tests", "s", false, "Enable Schema-based tests")
```